cmd/http-server/startcmd/start.go

```diff
@@ -317,10 +327,25 @@ func fetchTrustBlocWASMAgentOpts(cmd *cobra.Command) (*trustblocAgentJSOpts, err
 		return nil, err
 	}
 
+	// This is used for storage of docs in SDS.
+	// TODO get username from the actual registration process instead of a cmd line arg #266
+	agentUsername, err := cmdutils.GetUserSetVarFromString(
```

Comment:
is this field is mandatory?

Reply:
For now, yes. Each user will have their own vaults, so I need to know the username in order to differentiate between them. For example: starting up the demo with `make user-agent-start` starts two different agents, and both will store their DIDs in separate vaults. However, this command line arg is only temporary - eventually I'll be using the username from the user registration process instead of passing it in.